compiler-core/src/type_/tests.rs

```diff
@@ -234,15 +255,15 @@ pub fn compile_module(
     // place.
     let _ = modules.insert(PRELUDE_MODULE_NAME.into(), build_prelude(&ids));
 
-    for (name, module_src) in dep {
+    for (package, name, module_src) in dep {
```

Comment:
The dependencies now provide their own package name so that we can use it to simulate loading modules from different packages.